ibis/sql/alchemy.py

```diff
@@ -881,7 +939,17 @@ def _fetch(self, cursor):
             columns=cursor.proxy.keys(),
             coerce_float=True,
         )
-        return self.schema().apply_to(df)
+        df = self.schema().apply_to(df)
+        if GEO:
```

Comment:
We *may* want to move this logic elsewhere (possibly even to `Schema.apply_to`), but for now this is convenient.